packages/terra-dev-site/src/webpack/loaderUtils/generateNavigationConfig.js

```diff
@@ -302,7 +303,7 @@ const generatePagesConfig = ({
     if (item.additionalContent) {
       item.additionalContent.forEach((content) => {
         filePaths.push({
-          filePath: content.filePath,
+          filePath: formatPath(content.filePath),
           entryPoint: `/${content.label}.${item.contentExtension}${path.extname(content.filePath)}`,
```

Comment:
Do we also need to call formatPath on this `content.filePath`?

Reply:
We don't need it on line 307 because path.extname returns the extension correctly either way. It is only needed for the filePath property on line 306.